### ISSUE TITLE
renorm operator fails test conj_view 

### ISSUE THREAD

**Issue by Varal7** [pytorch/pytorch#59568]
## 🐛 Bug

Building on the current viable/strict (#59522) using recommended flags from [Contributing](https://github.com/pytorch/pytorch/blob/master/CONTRIBUTING.md) 

The test `test_conj_view_renorm_cpu_complex64 (__main__.TestCommonCPU)` fails.

## To Reproduce

1.  `git checkout 459270ac01f1bfcbeaffb20f1c94622561af94e0`
2.  `rm -rf build && BUILD_CAFFE2=0 USE_DISTRIBUTED=0 USE_MKLDNN=0 USE_CUDA=0 BUILD_TEST=0 USE_FBGEMM=0 USE_NNPACK=0 USE_QNNPACK=0 USE_XNNPACK=0 python setup.py develop`
3.  `TORCH_SHOW_CPP_STACKTRACES=1 python test/test_ops.py -v`

```
test_conj_view_renorm_cpu_complex64 (__main__.TestCommonCPU) ... python: ../c10/util/SmallVector.h:518:
c10::SmallVectorImpl<T>::iterator c10::SmallVectorImpl<T>::erase(c10::SmallVectorImpl<T>::const_iterator
) [with T = long int; c10::SmallVectorImpl<T>::iterator = long int*; c10::SmallVectorImpl<T>::const_iterator = const long int*]: Assertion `Iit >= this->begin() && "Iterator to erase is out of bounds."' failed.
```

## Expected behavior

Test does not fail.

## Environment

PyTorch version: 1.10.0a0+git459270a
Is debug build: True
CUDA used to build PyTorch: None
ROCM used to build PyTorch: N/A

OS: CentOS Stream 8 (x86_64)
GCC version: (GCC) 8.4.1 20200928 (Red Hat 8.4.1-1)
Clang version: 9.0.20190721
CMake version: version 3.18.2
Libc version: glibc-2.28

Python version: 3.8 (64-bit runtime)
Python platform: Linux-5.2.9-229_fbk15_hardened_4185_g357f49b36602-x86_64-with-glibc2.10
Is CUDA available: False
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
HIP runtime version: N/A
MIOpen runtime version: N/A

Versions of relevant libraries:
[pip] Could not collect
[conda] numpy                     1.20.3                   pypi_0    pypi
[conda] pytorch-sphinx-theme      0.0.24                    dev_0    <develop>
[conda] torch                     1.10.0a0+git459270a           dev_0    <develop>

## Additional context

cc @ezyang @gchanan @zou3519 @bdhirsh @jbschlosser @anjali411 @Varal7 @peterbell10

**Comment by albanD** [comment]
High pri because this set of compilation flags is the one recommended in the CONTRIBUTING guide. And it seems like they are not tested in our CI cc @malfet

**Comment by ngimel** [comment]
Can't repro using provided instructions. 
```
(pytorch) [ngimel@devgpu006.prn1 ~/local/pytorch] python -c "import torch; print(torch.__version__)"
1.10.0a0+git459270a
(pytorch) [ngimel@devgpu006.prn1 ~/local/pytorch] python test/test_ops.py -v -k test_conj_view_renorm_cpu_complex64
test_conj_view_renorm_cpu_complex64 (__main__.TestCommonCPU) ... ok

----------------------------------------------------------------------
Ran 1 test in 0.067s
```

**Comment by Varal7** [comment]
From personal correspondence, @albanD mentioned it would probably be due to my build flags (copied from [CONTRIBUTING](https://github.com/pytorch/pytorch/blob/master/CONTRIBUTING.md#build-only-what-you-need) )

I can rebuild with less flags to narrow down the issue.

**Comment by ngimel** [comment]
I've rebuilt with your build flags.

**Comment by Varal7** [comment]
Indeed, I tried to follow the instructions I provided and can not reproduce the issue either. 
I guess it is possible that I was using different build flags than the ones I provided. Unfortunately, I've already deleted the `build` directory of the misbehaving build and am not able to say what those flags were.
I'm closing this issue.

(As a side note, I still have errors when running `test_ops.py` due not missing LAPACK and will open a different issue for that. Doesn't sound as bad since we have the `skipIfNoLapack` decorator)
